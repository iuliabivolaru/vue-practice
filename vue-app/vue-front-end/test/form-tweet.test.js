import Vue from "vue";
import { shallowMount } from '@vue/test-utils'
import FormTweetComponent from '../src/views/form-tweet'

describe('form-tweet component', () => {
    const wrapper = shallowMount(FormTweetComponent);
    let $mounted = new Vue(FormTweetComponent).$mount();

    test('verify the component`s snapshot', () => {
        let $html = $mounted.$el.outerHTML
        expect($html).toMatchSnapshot();
    })
})