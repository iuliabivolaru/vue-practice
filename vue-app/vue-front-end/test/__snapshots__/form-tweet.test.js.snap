// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`form-tweet component verify the component\`s snapshot 1`] = `"<form action=\\"http://localhost:3000/posts\\" method=\\"postTweet\\"><div><input type=\\"text\\" placeholder=\\"What's your story?\\" class=\\"input is-rounded\\"></div> <!----> <div><textarea type=\\"text\\" placeholder=\\"Write some thoughts here...\\" class=\\"textarea is-rounded\\" style=\\"margin-top: 1%;\\"></textarea></div> <!----> <input type=\\"submit\\" value=\\"Post\\" class=\\"button is-normal is-pulled-right\\" style=\\"margin-top: 1%; margin-right: 1.9%;\\"></form>"`;
